__tests__/database.test.js

```diff
@@ -59,13 +58,48 @@ describe("GET endpoints", () => {
     });
 });
 
+
+describe("GET articles", () => {
+    test("Request from /api/articles/1 returns an object with required values", () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            expect(typeof body.author).toBe("string")
+            expect(typeof body.title).toBe("string")
+            expect(typeof body.article_id).toBe("number")
+            expect(typeof body.body).toBe("string")
+            expect(typeof body.topic).toBe("string")
+            expect(typeof body.created_at).toBe("string")
+            expect(typeof body.votes).toBe("number")
+            expect(typeof body.article_img_url).toBe("string")
+        })
+    })
+    test("Request from /api/articles/999999 when 999999 doesn't exist, respond with 404 Nothing found", () => {
```

Comment:
I would make this test description more general - it's a valid but non-existent article id